src/esp/assets/PTexMeshData.cpp

```diff
@@ -760,7 +759,7 @@ void PTexMeshData::parsePLY(const std::string& filename,
   Cr::Containers::Array<const char, Cr::Utility::Directory::MapDeleter>
       mmappedData = Cr::Utility::Directory::mapRead(filename);
 
-  const size_t fileSize = io::fileSize(filename);
+  const auto fileSize = *Cr::Utility::Directory::fileSize(filename);
```

Comment:
Could have stayed as `size_t`, I think?

(Argh, *why* I don't have it 64-bit always? :scream:)